lib/handler/proxy.c

```diff
@@ -96,6 +96,7 @@ static void on_context_init(h2o_handler_t *_self, h2o_context_t *ctx)
         client_ctx->websocket_timeout = NULL;
     }
     client_ctx->ssl_ctx = self->config.ssl_ctx;
+    client_ctx->ssl_session_cache = h2o_cache_create(0, 4096, 86400 * 1000, h2o_socket_ssl_destroy_session_cache_entry);
```

Comment:
In my understanding,  `on_context_init` is called for each thread, so this session cache never be accessed from other threads. Is this wrong?